src/libraries/System.Speech/src/Recognition/Grammar.cs

```diff
@@ -920,11 +918,7 @@ private void CreateSandbox(MemoryStream stream)
             {
                 // Check all methods referenced in the rule; availability, public and arguments
                 Assembly executingAssembly = Assembly.GetExecutingAssembly();
```

Comment:
This is the remaining location in the code where an AppDomain was removed. It was being used to cleanly unload compiled grammars. Since these can't be produced on .NET Core, and this is a (presumably) niche part of a legacy tech, I did not attempt to find a way to test this path (presumably using a canned grammar assembly created using .NET Framework) and rewrite it to use ALC. It should be functional as is.